src/custom-element.js

```diff
@@ -874,11 +874,10 @@ function createBaseAmpElementProto(win) {
   */
   ElementProto.getIntersectionChangeEntry = function() {
     const box = this.implementation_.getIntersectionElementLayoutBox();
-    const rootBounds = this.implementation_.getViewport().getRect();
-    return getIntersectionChangeEntry(
-        Date.now(),
-        rootBounds,
-        box);
+    const owner = this.resources_.getResourceForElement(this).getOwner();
+    const viewportBox = this.implementation_.getViewport().getRect();
+    const ownerBox = owner && owner.getLayoutBox();
```

Comment:
This is fine for now, but I think eventually we might need this to be recursive.